### PR TITLE
bpo-40208: Remove deprecated has_exec method of SymbolTable

### PR DESCRIPTION
**Pull request by isidentical**
https://bugs.python.org/issue40208

### CODE REVIEW

**Review by corona10 — CHANGES_REQUESTED** [review]
Please update Doc/library/symtable.rst also :)

**Comment by isidentical** [comment]
Thanks for the reminder. I have made the requested changes; please review again

**Review by corona10 — APPROVED** [review]
LGTM, I 've also checked @gvanrossum and @terryjreedy  's comment.

It looks like safe to remove :)
Thanks for catching this!

cc @vstinner

**Review by terryjreedy — APPROVED** [review]
Approved with minor change.  Will merge after tests pass again.
Corona10 war referring to comments on issue, in particular, Guido's approval.

**Comment by terryjreedy** [comment]
@corona10 Since you did initial review, go ahead and merge and close issue if you want.  I will wait until my tomorrow.

**Comment by corona10** [comment]
@terryjreedy 

Thanks :) Since I am post-promotion mentoring, 
I need to ask @vstinner before pressing the merge button. (https://discuss.python.org/t/vote-to-promote-dong-hee-na/3794)
So if you don't mind, would you like to wait for 2-3 days? 
but if the issue is urgent, it is okay to merge by you :)

Thank you for understanding :)

File: Doc/whatsnew/3.9.rst
Comment (vstinner) [inline Doc/whatsnew/3.9.rst]: I suggest to document the old behavior: mention that the deprecated method always returned False.

Comment (isidentical) [reply Doc/whatsnew/3.9.rst]: Added.

**Review by vstinner — APPROVED** [review]
LGTM. @corona10: You can merge it.

**Pull request merged**
